<a href="https://colab.research.google.com/github/sahintarkan/Data_assignments/blob/main/F1498_Statistics_Asignment2_.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# STATISTICS
### ASSIGNMENT (CONFIDENCE INTERVALS) 

### EXERCISE 1.

Suppose scores on exams in statistics are normally distributed with an unknown population mean and a population standard deviation of 3 points. A random sample of 36 scores is taken and gives a sample mean (sample mean score) of 68. Find a confidence interval estimate for the population mean exam score (the mean score on all exams).
Find a 90% confidence interval for the true (population) mean of statistics exam scores. 

### SOLUTION 1.

In [ ]:
import numpy as np
from scipy import stats
import matplotlib.pyplot as plt
import seaborn as sns

In [ ]:
sd = 3
n = 36
xbar = 68
alpha = 0.90

In [ ]:
sem = sd/np.sqrt(n)
sem

0.5

In [ ]:
stats.norm.interval(alpha=alpha,loc=xbar, scale=sem)

(67.17757318652427, 68.82242681347573)

We estimate with 90% confidence that the true population mean exam score for all statistics students is between 67.18 and 68.82.

### EXERCISE 2.

What is the normal body temperature for healthy humans? A random sample of 130 healthy human body temperatures provided by Allen Shoemaker7 yielded 98.25 degrees and standard deviation 0.73 degrees. 
Give a 99% confidence interval for the average body temperature of healthy people.

### SOLUTION 2.

In [ ]:
n1 = 130
xbar1=98.25
sd1=0.73
alpha1 = 0.99

In [ ]:
sem1 = sd1/np.sqrt(n1)
sem1

0.06402523540941313

In [ ]:
stats.norm.interval(alpha=alpha1,loc=xbar1, scale=sem1)

(98.08508192246582, 98.41491807753418)

Therefore the evidence does not indicate that the average body temperature for healthy humans is different from 98.6◦.

### EXERCISE 3.

The administrators for a hospital wished to estimate the average number of days required for inpatient treatment of patients between the ages of 25 and 34. A random sample of 500 hospital patients between these ages produced a mean and standard deviation equal to 5.4 and 3.1 days, respectively.
Construct a 95% confidence interval for the mean length of stay for the population of patients from which the sample was drawn.

### SOLUTION 3.

In [ ]:
n2 = 500
sd2 = 3.1
xbar2 = 5.4
alpha2=0.95

In [ ]:
sem2 = sd2/np.sqrt(n2)
sem2 

0.13863621460498696

In [ ]:
stats.norm.interval(alpha=alpha2,loc=xbar2, scale=sem2)

(5.12827801242126, 5.67172198757874)